cddl/concise-mid-tag.cddl

```diff
@@ -27,14 +27,10 @@ $tag-id-type /= uuid
 tag-version-type = integer .default 0
 
 module-metadata-type = {
-  ? 0 => module-name,
-  ? 1 => $module-type,
-  ? 2 => module-version,
-  ? 3 => module-vendor,
-  ? 4 => module-index,
+  ? 0 => element-name,
+  ? 1 => module-version,
```

Comment:
Why don't we go a step further and put version too in the element-name?

Reply:
We have had lengthy internal discussion about it. The argument for not doing it seems to be that it is a slippery slope as you could ask the same question about SVN and possibly others. The reasoning behind a logical composition based on element-name is the composition is the same 'thing' independent of versioning. If a SW update to a NIC could change it into a TPM (lets say) then it would make sense to regard version as an element-name ingredient.

Reply:
I guess turning your phone into a brick does not count here, right?